Code/Tools/AssetProcessor/native/resourcecompiler/rcjob.cpp

```diff
@@ -392,23 +381,6 @@ namespace AssetProcessor
             }
         }
 
-        // We will only continue once the fingerprint of the file stops changing
-        unsigned int fingerprint = AssetUtilities::GenerateFingerprint(builderParams.m_rcJob->m_jobDetails);
-        while (fingerprint != builderParams.m_rcJob->GetOriginalFingerprint())
-        {
```

Comment:
this feels like it was fixing a previous bug... or does this situation just not happen any longer?